src/Orleans.Core/Streams/PersistentStreams/Options/PersistentStreamProviderOptions.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Orleans.Configuration
 {
-    public class PersistentStreamOptions
+    public class StreamInitializationOptions
```

Comment:
Suggest StreamStartupOptions or StreamLifecycleOptions, as these are more related to how the provider is started/stopped than it's initialization.